manim/mobject/types/vectorized_mobject.py

```diff
@@ -1392,19 +1392,20 @@ def get_end_anchors(self) -> np.ndarray:
         nppcc = self.n_points_per_cubic_curve
         return self.points[nppcc - 1 :: nppcc]
 
-    def get_anchors(self) -> np.ndarray:
+    def get_anchors(self) -> typing.Iterable[np.ndarray]:
         """Returns the anchors of the curves forming the VMobject.
 
         Returns
         -------
-        np.ndarray
+        typing.Iterable[np.ndarray]
             The anchors.
         """
         if self.points.shape[0] == 1:
             return self.points
-        return np.array(
-            list(it.chain(*zip(self.get_start_anchors(), self.get_end_anchors()))),
-        )
+
+        s = self.get_start_anchors()
+        e = self.get_end_anchors()
+        return [s[i // 2] if i % 2 == 0 else e[i // 2] for i in range(len(s) + len(e))]
```

Comment:
Hey, it's just that if you modify the return values of either ```get_start_anchors()```, ```get_end_anchors()```, or ```get_anchors_and_handles()```, you would actually be modifying the ```points``` attribute of the VMobject. But for ```get_anchors()```, it returns a copy of the values from points. I think it'd make sense to unify those behaviors to avoid confusion like in #3185.

Reply:
Hey @JinchuLi2002! Not sure your comment really is an answer to my question; I think it would be a good idea to unify the behavior of `get_anchors` and the other getters. Do your current suggestion and my approach from above behave differently? At first glance I don't really think so -- and if so, I'd still prefer the solution via itertools that avoids a bunch of integer divisions / reminder calculations.

We could unify the behavior of the methods in a follow-up PR?